transcode/transcode.go

```diff
@@ -54,20 +60,23 @@ func RunTranscodeProcess(transcodeRequest TranscodeSegmentRequest, streamName st
 
 	outputs := []clients.OutputVideo{}
 
-	// Parse the manifest destination of the segmented output specified in the request
-	segmentedOutputManifestURL, err := url.Parse(transcodeRequest.SourceManifestURL)
-	if err != nil {
-		return outputs, segmentsCount, fmt.Errorf("failed to parse transcodeRequest.UploadURL: %s", err)
+	var targetURL *url.URL
+	var err error
+	if transcodeRequest.TargetURL != "" {
+		targetURL, err = url.Parse(transcodeRequest.TargetURL)
+	} else {
+		// TargetURL not defined in the request, fall back the source manifest dir
```

Comment:
This is actually for the backwards-compatibility of the `/api/transcode/file` endpoint. For the `/api/vod`, `TargetURL` is always there.

Reply:
Should we check that explicitly in the code, or perhaps replace the comment with that? I found your comment on github more useful than the comment in the code!

Reply:
Updated the comment.